### DISCUSSION TITLE
Welcome to java-angular-basic Discussions!

### DISCUSSION THREAD

**Discussion by marco76** [marco76/java-angular-basic#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by azseza** [comment]
Hello, I would like to thank you first for the content you've posted, it was all of big help to me. 
Any idea how to deploy a single packaged angular-spring project to azure, especialy with the azure-pipelines.yml config ? 
Thank you .
